docs/docs/spec.md

```diff
@@ -942,21 +942,10 @@ the output of the following operation:
     > `builder.add(null)`, it would find that it does.
     >
     > To solve this, we need a special case for substitution for null-exclusive
-    > type parameters like the one on `ImmutableList.Builder`. That special case
-    > needs to produce a type with a nullness operator other than `UNSPECIFIED`.
-    > One valid option is to produce `NO_CHANGE`; we happened to choose
-    > `MINUS_NULL`.
-    >
-    > The choice between `NO_CHANGE` and `MINUS_NULL` makes little difference
-    > for the parameter types of `ImmutableList.Builder`, but it can matter more
-    > for other APIs' *return types*. For example, consider `@NullMarked class
-    > Foo<E extends @Nullable Object>`, which somewhere uses the type
-    > [`FluentIterable<E>`]. `FluentIterable` has a method `Optional<E>
-    > first()`. Even when `E` is a type like `String UNION_NULL` (or `String
-    > UNSPECIFIED`), we know that `first().get()` will never return `null`. To
-    > surface that information to tools, we need to define our substitution rule
-    > to return `E MINUS_NULL`: If we instead used `E NO_CHANGE`, then the
-    > return type would look like it might include `null`.
+    > type parameters like the one on `ImmutableList.Builder`, one that avoids
+    > producing a type with nullness operator `UNSPECIFIED`. Since the special
```

Comment:
The two "one"s confused me. How about this?

```suggestion
    > type parameters (like the one on `ImmutableList.Builder`) that doesn't
    > produce a type with nullness operator `UNSPECIFIED`. Since the special
```